lab-jinho/model/note.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const uuid = require('node-uuid');
+
+module.exports = function(restaurantname, address) {
+  if (!restaurantname) throw new Error('expected restaurant name');
+  if (!address) throw new Error('expected address');
+
+  this.id = uuid.v1();
+  this.restaurantname = restaurantname;
+  this.address = address;
```

Comment:
Currently this file is called note.js, instead it should be something like restaurant.js. Consistent filenames are best practice- they should be specific to the code they contain.